seclang/directives.go

```diff
@@ -121,7 +121,7 @@ func directiveSecRule(w *coraza.Waf, opts string) error {
 }
 
 func directiveSecResponseBodyAccess(w *coraza.Waf, opts string) error {
-	w.ResponseBodyAccess = (strings.ToLower(opts) == "on")
```

Comment:
There is a helper that should be used: ```parseBoolean(opts)```, I forgot to use it

Reply:
I'll add it.